modules/cbQuestion/Builder.php

```diff
@@ -120,7 +120,13 @@
 $smarty->assign('sqlquery', $focus->column_fields['sqlquery']);
 $smarty->assign('qpagesize', $focus->column_fields['qpagesize']);
 $smarty->assign('qtype', empty($focus->column_fields['qtype']) ? 'Table' : $focus->column_fields['qtype']);
-$smarty->assign('typeprops', decode_html($focus->column_fields['typeprops']));
+
+if (empty($focus->column_fields['typeprops']) || $focus->column_fields['typeprops'] == "") {
```

Comment:
empty returns true if the variable == '' so you don't need the second part of the OR